CommonPackageConfig.cmake

```diff
@@ -63,6 +63,13 @@ if(CMAKE_MAJOR_VERSION LESS 3 OR NOT TARGET ${PROJECT_NAME})
   return()
 endif()
 
+# A project may want to include CommonCPack without having used common_library.
+# Returning in that case (Probably we don't event want to generate other
+# Config files since there's no library).
+if(NOT ${PROJECT_NAME}Targets)
```

Comment:
I just realized that in line 62 the check should be NOT TARGET ${PROJECT_NAME}_ALIAS. Then it would do what you want to do and it does what it was intended to do :)